css/types/basic-shape.json

```diff
@@ -55,17 +68,17 @@
         },
         "inset": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/basic-shape#inset()",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/basic-shape",
```

Comment:
I was a little confused by dropping the URL fragment here. Does the linked fragment not cover the right thing? If not, we could probably drop this subfeature mdn_url entirely, since the `__compat` links to the same place

Reply:
OK, seeing the other changes my question is answered. It'd be nice to see the inset fragment restored here, for consistency's sake